### PR TITLE
docs(tree-shaking): bundle minified problem

### PR DESCRIPTION
**Pull request by lumo10**
in my version of webpack 4.10.2 bundle comes already minified by default. I need to set the development mode

### CODE REVIEW

**Comment by Legends** [comment]
?

**Comment by montogeek** [comment]
This change is out of the scope of Tree shaking guide, it is also already well documented in other pages.
What do you think @EugeneHlushko @jeremenichelli ?

**Comment by Legends** [comment]
> If not set, webpack sets production as the default value for mode
[Section mode](https://webpack.js.org/concepts/mode/#usage)

**Comment by montogeek** [comment]
I guess it is important to be able to read the source code in order to make sure your tree shakeable modules are not there.

**Comment by montogeek** [comment]
Thanks

**Comment by Legends** [comment]
@lumo10 your last comment made it clear ;-)

File: src/content/guides/tree-shaking.md [hunk 1]
```diff
@@ -57,6 +58,24 @@ export function cube(x) {
...
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'dist')
+- }
++ },
++ mode: "development"
```
Comment (jeremenichelli) [inline src/content/guides/tree-shaking.md]: Sorry for being late to the party but, is it necessary to specify entry and output when they have default values already?

Also, I think that I would add the mode by CLI and put a small warning on the guide maybe.

Comment (lumo10) [reply src/content/guides/tree-shaking.md]: You are right if you take the single page guide alone, but in the contest of the get started guide where you  build the concept step by step you already have the webpack.config.js

**Pull request merged**